plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -83,14 +91,15 @@ function getCodeLinkInfo(entity: Entity): CodeLinkInfo {
 
 type AboutCardProps = {
   entity: Entity;
+  variant?: string;
 };
 
-export function AboutCard({ entity }: AboutCardProps) {
+export function AboutCard({ entity, variant }: AboutCardProps) {
   const classes = useStyles();
   const codeLink = getCodeLinkInfo(entity);
 
   return (
-    <Card>
+    <Card className={variant === 'gridItem' ? classes.gridItemCard : ''}>
```

Comment:
Am i right in understanding that this is an unfortunate hack because it doesn't use InfoCard? Pity to not import and reuse what the InfoCard implementation does (or, alternatively, if there was some way of injecting this behavior into `Card` that doesn't involve direct class manipulation, but i guess there isn't).

Approved anyway, let's get this in